src/getHumanAge.test.js

```diff
@@ -7,4 +7,60 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return 0 for both if input values equals 0', () => {
```

Comment:
```suggestion
  test('should return [0, 0] if input values are equal to 0', () => {
```
because in your case it seems like the function returns a number, but it's not true. fix everywhere in your test cases